packages/gnutls/SPECS/gnutls.spec

```diff
@@ -100,38 +159,71 @@ echo "SYSTEM=NORMAL" >> tests/system.prio
 # via the crypto policies
 
 %build
-export CCASFLAGS="$CCASFLAGS -Wa,--generate-missing-build-notes=yes"
-
-./configure --prefix=/usr/sgug \
-   --disable-option-checking \
-   --disable-doc \
-   --disable-hardware-acceleration \
-   --disable-padlock \
-   --disable-strict-der-time \
-   --disable-heartbeat-support \
-   --disable-tests \
-   --disable-full-test-suite \
-   --disable-gcc-warnings \
-   --disable-libdane \
-   --disable-nls \
-   --disable-rpath \
-   --enable-sha1-support \
-   --disable-static \
-   --disable-non-suiteb-curves \
-   --disable-guile \
-   --with-default-trust-store-pkcs11="pkcs11:" \
-   --libdir=%{?__isa_bits==64:/usr/sgug/lib}%{!?__isa_bits==32:/usr/sgug/lib32} \
```

Comment:
FYI, we have a macro for libdir - `%{_libdir}` - which is a bit simpler than the __isa_bits dance above .-)